charts/identity-platform/templates/platform-config-configmap.yaml

```diff
@@ -41,9 +41,12 @@ data:
   {{- end }}
   AM_STORES_USER_SERVERS: {{ $idrepoServers }}
   AM_SERVER_FQDN: {{ first .Values.platform.ingress.hosts }}
+  {{- if .Values.ds_restore.autoRestore }}
```

Comment:
Check out Paul's comment to your question in slack. You can use the quote function to handle this.

{{ .Values.ds_restore.autoRestore | quote }}